workshops/avd-lab-guide.md

```diff
@@ -269,7 +269,7 @@ At this point, you should be able to ping between hosts within a site but not be
 
 ## **Connect Sites to WAN IP Network**
 
-The WAN IP Network is defined by the `core_interfaces` data model. Full data model documentation is located [here](https://avd.sh/en/stable/roles/eos_designs/doc/core-interfaces-BETA.html).
+The WAN IP Network is defined by the `core_interfaces` data model. Full data model documentation is located [here](https://avd.sh/en/v4.1.0/roles/eos_designs/docs/input-variables.html#core-interfaces-settings).
```

Comment:
Then we won't have to update it in the future...

Reply:
Since these workshops usually refer to a specific version, I think its not a bad take to lock down the links to that version as well. That way we guarantee the links never break. The stable links as they were before, broke with the updates to the 4.0 docs as stable.

Reply:
If we do this, then we should update this section as well:
```
ansible-galaxy collection install arista.avd arista.cvp --force
export ARISTA_AVD_DIR=$(ansible-galaxy collection list arista.avd --format yaml | head -1 | cut -d: -f1)
pip3 config set global.disable-pip-version-check true
pip3 install -r ${ARISTA_AVD_DIR}/arista/avd/requirements.txt
```